src/drivers/cloudflare-kv-binding.ts

```diff
@@ -48,8 +49,12 @@ export default defineDriver((opts: KVOptions) => {
     },
     setItem(key, value, topts) {
       key = r(key);
+      let expirationOpts: TransactionOptions = {};
+      if (topts.ttl) {
+        expirationOpts.expirationTtl = topts.ttl;
+      }
       const binding = getKVBinding(opts.binding);
-      return binding.put(key, value, topts);
+      return binding.put(key, value, expirationOpts);
```

Comment:
We were already passing all topts. this is a breaking change since limits what we set now. (as proper todo, we should _transform_ `ttl` option to `expirationTtl`.